router/internal/trace/utils.go

```diff
@@ -20,7 +21,12 @@ func TracerFromContext(ctx context.Context) (tracer trace.Tracer) {
 }
 
 func RequestFilter(r *http.Request) bool {
-	if r.URL.Path == "/health" || r.URL.Path == "/favicon.ico" || r.Method == "OPTIONS" {
+
+	// /health and /health/live, /health/ready are ignored
+	if strings.HasPrefix(r.URL.Path, "/health") {
```

Comment:
can the user configure the health endpoint via config? if so, should we pick this up?